adcc/ReferenceState.py

```diff
@@ -155,7 +155,8 @@ def __init__(self, hfdata, core_orbitals=None, frozen_core=None,
             self.orbital_coefficients, self.conv_tol
         )
 
-        for name in ["excitation_energy_corrections"]:
+        self.solvent = None  # no solvent attached by default
+        for name in ["excitation_energy_corrections", "solvent"]:
```

Comment:
solvent -> solvation ?

Reply:
Maybe `embedding` or `environment` would be even better... I'd not call a protein modeled with PE "solvent" 😄

Reply:
Ok for both embedding or environment.

Reply:
Done.